### ISSUE TITLE
Markdown Formatting Wrong

### ISSUE THREAD

**Issue by teamayu** [pbek/QOwnNotes#1602]
Hi Patrizio,

This happened since the last update. Formatting for Markdown is lost for emphasised text. My color scheme settings are:

![1](https://user-images.githubusercontent.com/10624092/72958398-12785280-3de2-11ea-93af-3a2bcd4f1e35.jpg)

The display is:

![2](https://user-images.githubusercontent.com/10624092/72958399-12785280-3de2-11ea-97fc-8b0df2e7ed5e.jpg)

**Comment by pbek** [comment]
Thank you for reporting, I can reproduce that.
@Waqar144, another regression 😁

**Comment by Waqar144** [comment]
With the recent changes, it's kind of really hard to do that lol

**Comment by teamayu** [comment]
Glad you can reproduce the issue. Sorry for not using the reporting system, but didn't think it was needed in this case. :)

**Comment by Waqar144** [comment]
Alright, Its fixed (I think).

It now works like this:
- You will get the specified format if you are in plain text.
- If you are in a heading or a link or some place with color other than default text color, the color of the link/heading will be used.

**Comment by pbek** [comment]
## 20.1.15
- fixed stacked code block highlighting and improved non-fenced code block highlighting (thank you @Waqar144)

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

**Comment by teamayu** [comment]
Hi Patrizio,

There is no change, 20.1.14 is the same as the previous version.

**Comment by pbek** [comment]
we are talking about 20.1.15

**Comment by teamayu** [comment]
Oh, sorry. I downloaded it from the QOwnNotes website when you said there was a new version. The version that downloaded was 20.1.14. Let me try again.

**Comment by teamayu** [comment]
Ok, now I have 20.1.15 and that is working great. Thank you.

**Comment by pbek** [comment]
Great, thank you for testing!

**Comment by pbek** [comment]
> I downloaded it from the QOwnNotes website

doesn't the auto-update of QOwnNotes work for you? 😁
